ereadingtool/ereadingtool
# Upgrade merge words functionality

This PR upgrades merge words to handle cases where some words have not been added as a text word. This is done by sequencing POST tasks to add each text word before performing the merge.

To test this PR:

1. Open a text you have created as an instructor
2. Navigate to the translations tab
3. Select a word and add it as a text work
4. Click the Merge button and select and adjacent word that has not been added as a text word
5. Click on the Merge Together button
6. The words should be merged successfully

Related to #176.

This PR also removes legacy endpoints and an unused CSRF token flag.
